support/Pipelines/Bbh/PostprocessId.py

```diff
@@ -44,6 +52,8 @@ def postprocess_id(
       surface coordinates and coefficients are written to the 'horizons_file' in
       subfiles 'Ah{A,B}/Coordinates' and 'Ah{A,B}/Coefficients'.
 
+    - Alow user to trigger control of BBH parameters.
```

Comment:
Typo "allow". Add a few more details. Maybe something like "If 'control' is set to True, run a control loop such that masses and spins of the horizons match the input parameters. See ControlId.py for details.".